mqttclients/c/README.md

```diff
@@ -90,7 +93,7 @@ sudo apt install -y clang-format-9
 To fix any style errors, run this command from the root of the repo (if you used a different install method than apt install, use `clang-format` instead of `clang-format-9`):
 
 ``` bash
-clang-format-9 -style=file -i $(find . -name "*.[ch]" -not -path "./*/build/*")
+clang-format-9 -style=file -i $(find . -name "*.[ch]" -not -path "./*/build/*" -not -name "*.pb-c.*")
```

Comment:
Should the generated files also be excluded from the repo (and require re-generating)?

Reply:
They're now generated by the user and excluded from the repo, but the style checker should still exclude them since they'll probably be on the computer when running it